src/LEGO.AsyncAPI/Models/Bindings/Http/HttpMessageBinding.cs

```diff
@@ -16,12 +16,12 @@ public class HttpMessageBinding : IMessageBinding
         /// <summary>
         /// A Schema object containing the definitions for HTTP-specific headers. This schema MUST be of type object and have a properties key.
         /// </summary>
-        public AsyncApiSchema Headers { get; set; }
+        public AsyncApiSchema? Headers { get; set; }
 
         /// <summary>
         /// The version of this binding. If omitted, "latest" MUST be assumed.
         /// </summary>
-        public string BindingVersion { get; set; }
+        public string? BindingVersion { get; set; }
```

Comment:
This should not be null, as it should have a default value of "lastest"

Reply:
OK, I will set that for all places with BindingVersion.